test/unit/metrics/test_plot_metricframe.py

```diff
@@ -0,0 +1,184 @@
+# Copyright (c) Fairlearn contributors.
+# Licensed under the MIT License.
+
+import numpy as np
+import pytest
+from sklearn.metrics import recall_score, accuracy_score, confusion_matrix
+from fairlearn.experimental.enable_metric_frame_plotting import plot_metric_frame
```

Comment:
If we're doing sklearn style experimental module design, this would be 

```suggestion
from fairlearn.experimental import enable_metric_frame_plotting # noqa 
from fairlearn.metrics import plot_metric_frame
```

Reply:
This doesn't work, I get the error `ImportError: cannot import name 'plot_metric_frame' from 'fairlearn.metrics'`